packages/flutter_tools/lib/executable.dart

```diff
@@ -66,6 +67,7 @@ Future<Null> main(List<String> args) async {
     ..addCommand(new RunCommand())
     ..addCommand(new RunMojoCommand(hidden: !verboseHelp))
     ..addCommand(new ScreenshotCommand())
+    ..addCommand(new SkiaCommand())
```

Comment:
This is a feature intended for engine debugging and performance analysis, and I doubt it would be of interest to most app developers.

How do you think it should be categorized?  I don't know if there will be any more Skia-related commands, so maybe we should create a top level command for miscellaneous debugging utilities?